### ISSUE TITLE
ProtocolError: Runtime.callFunctionOn timed out.

### ISSUE THREAD

**Issue by Beluk** [jean-humann/docs-to-pdf#22]
![image](https://github.com/jean-humann/docs-to-pdf/assets/10911904/a19d59f7-e28d-4a3b-8435-c63168fdb496)

**Comment by jean-humann** [comment]
Hi @Beluk , could you please provide a way to reproduce the Error, you had for me to be sure to be on the right way to develop a fix ?

**Comment by Beluk** [comment]
Hi @jean-humann,
it happens only on pdf which has a lot of pages. Find attached an example...

- npm start
- npx docs-to-pdf --initialDocURLs="http://localhost:3000/docs/intro" --contentSelector="article" --paginationSelector=".pagination-nav__link--next" --excludeSelectors=".margin-vert--xl a" --coverTitle="ProtocolError" --outputPDFFilename protocolerrror.pdf

[protocolerror.zip](https://github.com/jean-humann/docs-to-pdf/files/11495678/protocolerror.zip)

**Comment by jean-humann** [comment]
Fixed in [PR#26](https://github.com/jean-humann/docs-to-pdf/pull/26)
This PR add the option `--protocolTimeout` that user can set in the CLI.

@Beluk you can try the following command after update the version 👍 

``` bash
npx docs-to-pdf --initialDocURLs="http://localhost:3000/docs/intro" --contentSelector="article" --paginationSelector=".pagination-nav__link--next" --excludeSelectors=".margin-vert--xl a" --coverTitle="ProtocolError" --outputPDFFilename protocolerrror.pdf --protocolTimeout "500000"
```

**Comment by Beluk** [comment]
With the new version i'm getting the following error:
```
Der Befehl "docs-to-pdf" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
```

**Comment by Beluk** [comment]
@jean-humann 
somehow it cannot find docs-to-pdf, but mr-pdf is working... Here is my output of your command...
![image](https://github.com/jean-humann/docs-to-pdf/assets/10911904/531235ab-8a33-463e-9ec3-2b3be6d0a4b6)

**Comment by jean-humann** [comment]
Fixed in v0.2.4 
<img width="752" alt="Capture d’écran 2023-05-23 à 18 59 37" src="https://github.com/jean-humann/docs-to-pdf/assets/76565654/a257231f-e88b-441a-99c9-cd5a3bf6444d">
